tardis/plasma/assembly/base.py

```diff
@@ -0,0 +1,551 @@
+import logging
+
+import numpy as np
+import pandas as pd
+from astropy import units as u
+
+from tardis.plasma import BasePlasma
+from tardis.plasma.base import PlasmaSolverSettings
+from tardis.plasma.exceptions import PlasmaConfigError
+from tardis.plasma.properties import (
+    HeliumNumericalNLTE,
+    IonNumberDensity,
+    IonNumberDensityHeNLTE,
+    LevelBoltzmannFactorNLTE,
+    MarkovChainTransProbsCollector,
+    RadiationFieldCorrection,
+    StimulatedEmissionFactor,
+)
+from tardis.plasma.properties.base import TransitionProbabilitiesProperty
+from tardis.plasma.properties.level_population import LevelNumberDensity
+from tardis.plasma.properties.nlte_rate_equation_solver import (
+    NLTEPopulationSolverLU,
+    NLTEPopulationSolverRoot,
+)
+from tardis.plasma.properties.property_collections import (
```

Comment:
For clarity, could we `import property_collections as pc` or something like that?  This is a large file and it isn't immediately clear where the variables come from all the time

Reply:
as I said - I'm in the process of rewriting large chunks of this.